File: SAMPLE/SQL/GO/blog_type.go
type SECTION struct {
    Id uint64    `db:"Id"`;
    Number uint64    `db:"Number"`;
    Name string    `db:"Name"`;
    Text string    `db:"Text"`;
    Image string    `db:"Image"`;
    ImageIndex uint64    `db:"-"`;
}

// ~~

type USER struct {
    Id uint64    `db:"Id"`;
    FirstName string    `db:"FirstName"`;
    LastName string    `db:"LastName"`;
    Email string    `db:"Email"`;
    Pseudonym string    `db:"Pseudonym"`;
    Password string    `db:"Password"`;
    Phone string    `db:"Phone"`;
    Street string    `db:"Street"`;
    City string    `db:"City"`;
    Code string    `db:"Code"`;
    Region string    `db:"Region"`;
    Country string    `db:"Country"`;
    Company string    `db:"Company"`;
    ItIsAdministrator bool    `db:"ItIsAdministrator"`;
}

// ~~

type ARTICLE struct {
    Id uint64    `db:"Id"`;
    SectionId uint64    `db:"SectionId"`;
    UserId uint64    `db:"UserId"`;
    Title string    `db:"Title"`;
    Text string    `db:"Text"`;
    Image string    `db:"Image"`;
    Date time.Time    `db:"Date"`;
    Section * SECTION    `db:"-"`;
    User * USER    `db:"-"`;
    ImageIndex uint64    `db:"-"`;
}

// ~~

type COMMENT struct {
    Id uint64    `db:"Id"`;
    ArticleId uint64    `db:"ArticleId"`;
    UserId uint64    `db:"UserId"`;
    Text string    `db:"Text"`;
    DateTime time.Time    `db:"DateTime"`;
    Article * ARTICLE    `db:"-"`;
    User * USER    `db:"-"`;
}

// ~~

type SUBSCRIBER struct {
    Id uint64    `db:"Id"`;
    Name string    `db:"Name"`;
    Email string    `db:"Email"`;
}

File: SAMPLE/SQL/JS/blog_type.js
class SECTION
{
    constructor(
        )
    {
        this.Id = null;
        this.Number = null;
        this.Slug = null;
        this.Name = null;
        this.Text = null;
        this.Image = null;
        this.ImageIndex = null;
    }
}

// ~~

class USER
{
    constructor(
        )
    {
        this.Id = null;
        this.FirstName = null;
        this.LastName = null;
        this.Email = null;
        this.Pseudonym = null;
        this.Password = null;
        this.Phone = null;
        this.Street = null;
        this.City = null;
        this.Code = null;
        this.Region = null;
        this.Country = null;
        this.Company = null;
        this.ItIsAdministrator = null;
    }
}

// ~~

class ARTICLE
{
    constructor(
        )
    {
        this.Id = null;
        this.SectionId = null;
        this.UserId = null;
        this.Slug = null;
        this.Title = null;
        this.Text = null;
        this.Image = null;
        this.Date = null;
        this.Section = null;
        this.User = null;
        this.ImageIndex = null;
    }
}

// ~~

class COMMENT
{
    constructor(
        )
    {
        this.Id = null;
        this.ArticleId = null;
        this.UserId = null;
        this.Text = null;
        this.DateTime = null;
        this.Article = null;
        this.User = null;
    }
}

// ~~

class SUBSCRIBER
{
    constructor(
        )
    {
        this.Id = null;
        this.Name = null;
        this.Email = null;
    }
}
